tests/pipeline/test_pipeline_algo.py

```diff
@@ -361,87 +339,77 @@ def spot_price(self, sid, day, column):
         return 100.0
 
 
-class PipelineAlgorithmTestCase(TestCase):
+class PipelineAlgorithmTestCase(WithAdjustmentReader,
+                                ZiplineTestCase):
+    AAPL = 1
+    MSFT = 2
+    BRK_A = 3
+    assets = AAPL, MSFT, BRK_A
 
     @classmethod
-    def setUpClass(cls):
-        cls.AAPL = 1
-        cls.MSFT = 2
-        cls.BRK_A = 3
-        cls.assets = [cls.AAPL, cls.MSFT, cls.BRK_A]
-        asset_info = make_simple_equity_info(
+    def make_equity_info(cls):
+        return make_simple_equity_info(
```

Comment:
See note above re: making this pattern the default behavior via parameters.  Might also include an optional `ASSET_FINDER_EQUITY_SYMBOLS` for this case.